packages/beacon-node/src/chain/rewards/syncCommitteeRewards.ts

```diff
@@ -0,0 +1,58 @@
+import {CachedBeaconStateAllForks, CachedBeaconStateAltair} from "@lodestar/state-transition";
+import {ValidatorIndex, allForks, altair} from "@lodestar/types";
+import {ForkName, SYNC_COMMITTEE_SIZE} from "@lodestar/params";
+import {routes} from "@lodestar/api";
+
+export type SyncCommitteeRewards = routes.beacon.SyncCommitteeRewards;
+
+export async function computeSyncCommitteeRewards(
+  block: allForks.BeaconBlock,
+  preState: CachedBeaconStateAllForks,
+  filters?: (ValidatorIndex | string)[]
+): Promise<SyncCommitteeRewards> {
+  const fork = preState.config.getForkName(block.slot);
+  if (fork === ForkName.phase0) {
+    throw Error("Cannot get sync rewards as phase0 block does not have sync committee!");
```

Comment:
![Alt Text](https://media1.tenor.com/m/iDlNhrOpf8gAAAAC/yelling-why-are-you-yelling-at-me.gif)

```suggestion
    throw Error("Cannot get sync rewards as phase0 block does not have sync committee");
```

Reply:
> 
> 
> ![Alt Text](https://media1.tenor.com/m/iDlNhrOpf8gAAAAC/yelling-why-are-you-yelling-at-me.gif)
> 
> 

LOL